types.go

```diff
@@ -0,0 +1,17 @@
+package celeriac
+
+import "time"
+
+type Message struct {
+	Expiration string    // implementation use - message expiration spec
+	MessageID  string    // application use - message identifier
+	Timestamp  time.Time // application use - message timestamp
+	Type       string    // application use - message type name
+
+	DeliveryTag uint64
```

Comment:
How transferable are these fields with other brokers? Should this be a bit more primitive (Expiration, MessageID, Timestamp,  Body)?

Reply:
Yeah probably not very. Will remove